internal/clients/http/gcp/gcp_client.go

```diff
@@ -48,30 +48,30 @@ func (c *gcpClient) RunInstances(ctx context.Context, projectID string, namePatt
 		Zone:    zone,
 		BulkInsertInstanceResourceResource: &computepb.BulkInsertInstanceResource{
 			NamePattern: namePattern,
-			Count:       ptr.Int64(amount),
-			MinCount:    ptr.Int64(amount),
+			Count:       ptr.ToInt64(amount),
+			MinCount:    ptr.ToInt64(amount),
```

Comment:
Just a sidenote: You can just use `ptr.To` and generics will figure out the type for you :-)

You only need to use `ToInt64` when the type is `int` and not `int64`, like when you use type inference.

Let's just leave this, no big deal.

Reply:
or you can just `&amount` :P which would also do it :shrug:

Reply:
Lol that's right. Busted, for the second time today.